src/components/Capture/withCrossDeviceWhenNoCamera.tsx

```diff
@@ -42,11 +42,12 @@ const withCrossDeviceWhenNoCamera = <P extends CaptureComponentProps>(
         hasCamera,
         requestedVariant,
         step,
-      } = this.props
-
+        photoCaptureFallback = true,
+      } = { ...this.props }
```

Comment:
Sorry for being harsh here, but this `{ ...this.props}` will create an unnecessary copy step.

Reply:
no worries! I'll amend the last commit with this update

Reply:
When removing this copy step I get the following error: `Property 'photoCaptureFallback' does not exist on type 'CaptureComponentProps & Readonly<Attributes & { children?: ComponentChildren; ref?: Ref<any> | undefined; }>'`

I'm very much a beginner in TS, could you help me understand why this might be? I can see that 
```
type CaptureComponentProps =
  | StepComponentDocumentProps
  | (StepComponentFaceProps & { forceCrossDevice: never })
```
So why can't the property that is inside `StepComponentFaceProps` be reached?

Reply:
@ethan605 I updated the type that was throwing the error above and it's now working - but I'm not sure that's the correct way to do this. Do you mind taking a look? 🙏

Reply:
Sorry @natasharsilva, I couldn't take a look at this yesterday. The error was because you were trying to call `photoCaptureFallback` from `this.props`, which could be either `StepComponentDocumentProps` or `StepComponentFaceProps`. But because `photoCaptureFallback` belongs to `StepComponentFaceProps` only, it complained that in some situations when `this.props` has the type of `StepComponentDocumentProps`, then there would be no `photoCaptureFallback` at all.

To fix this, you should change the definition of `CaptureComponentProps` to:

```typescrtipt
type CaptureComponentProps =
  | (StepComponentDocumentProps & { photoCaptureFallback: never })
  | (StepComponentFaceProps & { forceCrossDevice: never })
```

In the latest commit, you put `photoCaptureFallback: true` which is not wrong, but also not reasonable for an absent property at all.

Reply:
Makes much more sense now, thanks! I changed it to `never` now along with some changes to try and help with the code readability :)